From be346ed14d158386f72e97c30ec91b82f6362270 Mon Sep 17 00:00:00 2001
From: Eric <eric.daigle@zaclys.net>
Date: Fri, 11 Jun 2021 21:00:38 -0400
Subject: [PATCH] dailyForecast.js: fix low temperature

As described in #190, an apparent typo currently causes the low temperatures
in the daily forecast view to be incorrect (not actually the lowest temperature for the period).

This MR corrects the updating of minTemp, fixing the low temperature.

Closes #190
---
 src/app/dailyForecast.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/app/dailyForecast.js b/src/app/dailyForecast.js
index 0bd94d5..b5d87e5 100644
--- a/src/app/dailyForecast.js
+++ b/src/app/dailyForecast.js
@@ -130,7 +130,7 @@ var DailyForecastBox = GObject.registerClass(class DailyForecastBox extends Gtk.
             }
             if (temp < minTemp) {
                 minInfo = info;
-                minTemp = minTemp;
+                minTemp = temp;
             }
 
             let datetime = Util.getDateTime(info);
-- 
GitLab

